lib/frontier/factory_girl_support/declaration.rb

```diff
@@ -7,8 +7,8 @@ def initialize(verb, factory_object)
     @factory_object = factory_object
   end
 
-  def to_s
-    "FactoryGirl.#{verb}(:#{class_name})"
+  def to_s(factory_options={})
+    "FactoryGirl.#{verb}(#{factory_body(factory_options)})"
```

Comment:
> Is this for generating premade factories? i dont really undestand what this is at all?

Check the specs :) It's generating a string that when injected inside a template will be generated as ruby code. That ruby code will evaluate to be a call to FactoryGirl